### PR TITLE
image-upload capability in editor 

### PR DESCRIPTION
**Pull request by rizwans-cuelogic**
1.image-upload capability in editor 
2.changes banner image field in edit form and create form
3.changes save draft and save

### CODE REVIEW

File: project4/project/ebs/static/ebs/css/add-blog.css [hunk 1]
```diff
@@ -1,3 +1,18 @@
+#preview img{
+	border: #dcdcdc 1px solid;
+    border-radius: 3px;
+    padding:3px;
+    width:100%;
+}
+#preview{
+	position:relative;
+}
+.fa-trash-o{
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/add-blog.css]: add color css to the .media-button class

File: project4/project/ebs/static/ebs/css/add-blog.css [hunk 2]
```diff
@@ -34,8 +49,14 @@
 	text-align:right;
 }
 .media-button{
+	vertical-align:bottom;
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/static/ebs/css/add-blog.css]: remove vertical-align, as it is not required when the element is at absolute position.

File: project4/project/ebs/templates/ebs/update_blog.html [hunk 3]
```diff
@@ -33,36 +34,19 @@ <h3 class="header">EDIT BLOG</h3>
...
-                </p>
-        {% endif %}
-        {% if image2_value %}
-            <p class="media-file" id="file3">
-            <button class="media-button pull-right" id="button3" type="button" value="{{image2_id}}"><i class="fa fa-trash-o" aria-hidden="true"></i></button>
-            {{image2_value}}
-            </p>
-        {% endif %} 
+           {% if attachments_value %}
+            <div id="preview" class="col-md-2">
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/update_blog.html]: change the bootstrap class name to align the div on right side.

**Comment by sagarp-cuelogic** [comment]
- remove border-radius form .navbar-default class and keep border only on bottom.
 - user profile drop-down left alignment is not proper.
 - there is 1px spacing below footer on edit blog page and not on another pages.
 - remove .text-right class form copyright section and use only .copyright class to align text.
 - Blog image div's right alignment is not proper.

File: project4/project/ebs/static/ebs/js/login.js [hunk 4]
```diff
@@ -52,6 +52,7 @@ $(document).ready(function() {
                 else{
                      $('#signinmodal').modal('hide');
                      $('#logged-user').text("Welocome "+result['user']);
+                     $('#dropdownMenu1').text("Welocome "+result['user']);
```
Comment (ananthu-nair707) [inline project4/project/ebs/static/ebs/js/login.js]: Welcome spelling.

File: project4/project/ebs/views.py [hunk 5]
```diff
@@ -285,17 +285,14 @@ def manage_blog(request):
                 'blog_title':each.title,
                 'blog_description':each.description
             }
-            files=BlogFile.objects.filter(blog_id=each.id).order_by('-id')
+            files=BlogFile.objects.filter(blog_id=each.id)
             if files is None:
                 context['blog_file']=''
             else:
                 for each in files:
                     print each.attachments
```
Comment (ananthu-nair707) [inline project4/project/ebs/views.py:292]: Remove debug/print statements

File: project4/project/ebs/views.py [hunk 6]
```diff
@@ -439,22 +404,18 @@ def detail_blog(request,id):
                 'related_timestamp':each.timestamp,
                 'related_categories':each.categories
             }
-        files=BlogFile.objects.filter(blog_id=each.id).order_by('-id')
+        files=BlogFile.objects.filter(blog_id=each.id)
         if not files :
             continue
         else:
             for each in files:
                 print each.attachments
```
Comment (ananthu-nair707) [inline project4/project/ebs/views.py]: Remove debug/print statements

**Pull request merged**